cli/lsp/testing/execution.rs

```diff
@@ -991,14 +947,5 @@ mod tests {
           .to_string()
       ]
     );
-    assert_eq!(
```

Comment:
Why remove a test assertion?  Specifically testing the serialization of JSON, where else is this tested?

Reply:
`.as_test_options()` doesn't exist anymore, filters aren't passed to JS since it's moved to Rust